src/bluesearch/database/article.py

```diff
@@ -520,6 +521,40 @@ def _element_to_str(self, element: Element | None) -> str:
             return self._inner_text(element)
 
 
+class MecaParser(JATSXMLParser):
+    """Parser for .meca files.
+
+    This could be used for articles from bioRxiv, and medRxiv.
+
+    Parameters
+    ----------
+    path
+        The path to a .meca file.
+    """
+
+    def __init__(self, path: str | Path) -> None:
+
+        with ZipFile(path) as myzip:
+            xml_files = [
+                x
+                for x in myzip.namelist()
+                if x.startswith("content/") and x.endswith(".xml")
+            ]
+
+            if len(xml_files) != 1:
+                raise ValueError(
+                    "There needs to be exactly one .xml file inside of content/"
+                )
+
+            xml_file = xml_files[0]
+
+            # Parsing logic
+            with myzip.open(xml_file, "r") as f:
+                self.content = ElementTree.parse(f)
+
+        self.ids = self.get_ids()
```

Comment:
I guess we're skipping `super().__init__(...)` on purpuse, otherwise this doesn't work? Not the best practise, but I guess there's no other way? I'd at least leave a comment to explain that we're aware of this and why this was done.